tests/TestAxis.cpp

```diff
@@ -137,3 +137,36 @@ TEST_CASE("set number of ticks", "[axis]") {
   CHECK(ax->get_ticks()[0] == 10);
   CHECK(ax->get_ticks()[1] == 10);
 }
+
+
+TEST_CASE("ylabel position: case 1", "[axis]") {
+  std::vector<double> predicted_data = {
```

Comment:
rename these vectors x and y and use these below rather than creating another pair of vectors